set.mm

```diff
@@ -641865,10 +641865,29 @@ D Fn ( I ... ( M - 1 ) ) /\
     ifexd.1 $e |- ( ph -> A e. V ) $.
     ifexd.2 $e |- ( ph -> B e. W ) $.
     $( Conditional operator existence.  (Contributed by SN, 26-Jul-2024.) $)
-    ifexd $p |- ( ph -> if ( ph , A , B ) e. _V ) $=
-      ( wcel cif cvv ifexg syl2anc ) ABDHCEHABCIJHFGABCDEKL $.
+    ifexd $p |- ( ph -> if ( ps , A , B ) e. _V ) $=
+      ( wcel cif cvv ifexg syl2anc ) ACEIDFIBCDJKIGHBCDEFLM $.
   $}
 
+  $( Move a conditional outside a relation.  (Contributed by SN,
+     14-Aug-2024.) $)
+  brif1 $p |- ( if ( ph , A , B ) R C <-> if- ( ph , A R C , B R C ) ) $=
+    ( cif wbr iftrue breq1d wn iffalse casesifp ) AABCFZDEGBDEGCDEGAMBDEABCHIAJ
+    MCDEABCKIL $.
+
+  $( Move a conditional outside a relation.  (Contributed by SN,
+     14-Aug-2024.) $)
+  brif2 $p |- ( C R if ( ph , A , B ) <-> if- ( ph , C R A , C R B ) ) $=
+    ( cif wbr iftrue breq2d wn iffalse casesifp ) ADABCFZEGDBEGDCEGAMBDEABCHIAJ
+    MCDEABCKIL $.
+
+  $( Move two conditionals outside a relation.  (Contributed by SN,
+     14-Aug-2024.) $)
```

Comment:
"inside and outside" (to emphasize it's a biconditional: you can go both ways).

Actually, it seems more natural to write it the other way:
> Move a relation inside and outside the conditional operator.